packages/osc-ecommerce/app/components/CurrencySelector/CurrencySelector.tsx

```diff
@@ -0,0 +1,64 @@
+import type { FormEvent } from 'react';
+import { Icon, Select, SelectItem } from 'osc-ui';
+import { Form, useLocation } from '@remix-run/react';
+import type { I18nLocale } from '~/types/shopify';
+import './currency-selector.css';
+
+interface CurrencySelectorProps {
+    classNames?: string;
+    region: I18nLocale;
+}
+
+export const CurrencySelector = (props: CurrencySelectorProps) => {
+    const { classNames, region } = props;
+
+    const { pathname } = useLocation();
+
+    const CURRENCY_OPTIONS = [
+        {
+            description: { label: 'Euro' },
+            value: 'FR',
+            country: 'en-fr',
+            name: <Icon id="euro" />,
+        },
+        {
+            description: { label: 'Sterling' },
+            value: 'GB',
+            country: 'en-gb',
+            name: <Icon id="sterling" />,
```

Comment:
I'm not sure these icons really work, they look a bit odd imo as they're a different font. Could you do something like this instead?
```suggestion
            name: <span className="t-font-m u-text-med">£</span>,
```
I think that looks a bit more consistent since it uses our primary font instead

Reply:
Ah yeah, good shout, didn't think of that approach!